go.mod

```diff
@@ -79,6 +79,7 @@ require (
 	github.com/josharian/intern v1.0.0 // indirect
 	github.com/json-iterator/go v1.1.12 // indirect
 	github.com/kr/pretty v0.2.1 // indirect
+	github.com/kubernetes-sigs/kubebuilder v1.0.8 // indirect
```

Comment:
where do we import this?

Reply:
wrong import while fix integration test, will remove